modules/database/main.tf

```diff
@@ -31,9 +31,10 @@ resource "google_sql_database_instance" "default" {
   deletion_protection = var.deletion_protection
 
   settings {
-    tier              = var.tier
-    availability_type = var.availability_type
-    user_labels       = var.labels
+    tier                        = var.tier
+    availability_type           = var.availability_type
+    user_labels                 = var.labels
+    deletion_protection_enabled = true
```

Comment:
One other thing, we should probably have the value here use `var.deletion_protection`. I don't see a scenario where we'd want one enabled without the other so using the same variable should be ok.

Reply:
fair enough, I'll update that.